source/_integrations/weather.darksky.markdown

```diff
@@ -61,6 +61,11 @@ mode:
   required: false
   type: string
   default: hourly
+scan_interval:
```

Comment:
Please remove this option, since it is already documented on the platform level.

<https://www.home-assistant.io/docs/configuration/platform_options/>

Thanks!

Reply:
The note at the top of the page linked says that the _"These options are being phased out"_ and it is in other integrations such as [Luftdaten](https://www.home-assistant.io/integrations/luftdaten/)

Reply:
Luftdaten is not a platform, but an integration.

Please adjust as requested.

Reply:
Weather Darksky is also an integration, isn't it?  It is under the `_integrations`

Reply:
Not sure why you try to fight this. Yes, it hasn't always be done right in the past, unfortunately. However, it does not give a reason to do it again.

Again, please adjust. This PR isn't merged in this state.

Reply:
I am not trying to be argumentative; I am failing to understand the reasoning.  I am assuming [this is an integration as it is currently listed as such](https://www.home-assistant.io/integrations/weather.darksky/).  Not all integrations utilize `scan_interval` and Weather Darksky currently is one that does not.  This is why (there is [a corresponding PR in core)[https://github.com/home-assistant/core/pull/32589].  If this is a platform, then this PR is moot as the document needs to be removed. Additionally, platform page does linked page does say at the very top 

> "These options are being phased out and are only available for single platform integrations." 

If I am to utilize the platform options over single platform integrations, can you please link where this is stated?

Reply:
It is implemented as a platform in the codebase, thus has the platform schema (which you know, because you didn't need to change the configuration schema either).

Hence, the configuration option is inherited for this platform integration, which is linked. There is no scan interval in your schema because of that.

<https://github.com/Thrilleratplay/core/blob/weather_darksky/set_scan_interval/homeassistant/components/darksky/weather.py#L65>

Please don't confuse a component integration with a platform integration.

Reply:
Thank you for the explanation and your patience.  The subtleties are incredibly confusing to those new to the project.  I will either update or withdraw this PR depending on the changes required in the core PR.

Reply:
Yes, I agree. Most things were platforms in the past. We are slowly moving away from that (more and more platforms are converted to full component integrations). Furthermore, we are reducing the use of scan interval in general, since we should supply sane values out of the box.